pkg/test/framework/components/ingress/kube.go

```diff
@@ -80,7 +80,16 @@ func getHTTPAddressInner(env *kube.Environment, ns string) (interface{}, bool, e
 			return nil, false, fmt.Errorf("no ports found in service: %s/%s", ns, "istio-ingressgateway")
 		}
 
-		port := svc.Spec.Ports[0].NodePort
+		var port int32 = 1
```

Comment:
nit: why not 0?

Reply:
that should be a false-alarm
`should drop = 0 from declaration of var inst; it is the zero value go-lint`

Reply:
Right, that means that this line should be `var port int32`. You don't need the `= 0` because go automatically defaults variables to their zero value.

Reply:
Thanks @nmittler for your comments.